setup.py

```diff
@@ -308,6 +308,9 @@ def run_tests(self):
 
     install_requires=requirements,
     tests_require=test_requirements,
+    extras_require={
```

Comment:
I don't know. As you said, the only place we use this is development workflow, so I don't care :-)